### ISSUE TITLE
fine-tuning

### ISSUE THREAD

**Issue by stethbo** [mrutkowsky/NAWROS#6]
Fine tune BERT on GPU  provided by the unversity.

Subtasks:
* prepare dataset
* build dataloader
* prepare model for finetuning
* build trainingloop with appropriate loss functions and hyperparameters
